tuner/tuner/common.py

```diff
@@ -38,10 +37,24 @@ def getI64(self, value: int) -> ir.IntegerAttr:
 
 
 class TunerContext:
-    def __init__(self, mlir_ctx: ir.Context, logger: logging.Logger):
-        self.mlir_ctx: ir.Context = mlir_ctx
-        self.logger: logging.Logger = logger
-        self.type: CommonTypes = CommonTypes(mlir_ctx)
+    def __init__(self, logger: Optional[logging.Logger] = None):
+        self.mlir_ctx: ir.Context = ir.Context()
+        self.logger: logging.Logger = logger or logging.getLogger(
+            "tune"
+        )  # Default to "tune" logger
```

Comment:
```suggestion
        self.logger: logging.Logger = logger or logging.getLogger("tune")
```

I don't see how this comments helps